library/Zend/Mail/Message.php

```diff
@@ -36,6 +36,44 @@ class Message
     protected $encoding = 'ASCII';
 
     /**
+     * Constructor.
+     *
+     * @param null|array|Traversable $options (Optional) Options to set
+     * @param array|Traversable $options
+     */
+    public function __construct($options = null)
+    {
+        if (null !== $options) {
+            $this->setOptions($options);
+        }
+    }
+
+    /**
+     * @param  array|Traversable $options         Options to set
+     * @return self
+     * @throws Exception\InvalidArgumentException
+     */
+    public function setOptions($options)
```

Comment:
I'm asking for the method to be `private` unless there is any reason for it to be `public`